.circleci/config.yml

```diff
@@ -1,15 +1,19 @@
+---
 version: 2
 jobs:
   build:
+    working_directory: ~/useradm
     docker:
-      - image: python:3.5
-    working_directory: /home/ubuntu/useradm
+      - image: python:3-alpine
+        environment:
+          ENV: CI
+      - image: dinkel/openldap
```

Comment:
docker-openldap can load an ldif.

`docker exec $LDAP_CID ldapadd` after the container is up.

You can also extend (bootstrap) the image using a regular [docker image file](https://github.com/osixia/docker-openldap/blob/stable/example/extend-osixia-openldap/bootstrap/ldif/billy.ldif).